### PR TITLE
Fix package downgrade warnings; fix unit tests

### PR DESCRIPTION
**Pull request by loic-sharma**

### CODE REVIEW

File: src/NuGet.Services.Revalidate/NuGet.Services.Revalidate.csproj [hunk 1]
```diff
@@ -117,10 +117,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="NuGet.Services.Status">
-      <Version>2.27.0</Version>
+      <Version>2.28.0</Version>
```
Comment (loic-sharma) [inline src/NuGet.Services.Revalidate/NuGet.Services.Revalidate.csproj]: NuGet.Services were updated to 2.28.0, but this project was missed.

File: src/NuGet.SupportRequests.Notifications/Tasks/SupportRequestsNotificationScheduledTask.cs [hunk 2]
```diff
@@ -15,8 +15,6 @@ internal abstract class SupportRequestsNotificationScheduledTask<TNotification>
       : IScheduledTask
         where TNotification : INotification
     {
-        private InitializationConfiguration _configuration;
```
Comment (loic-sharma) [inline src/NuGet.SupportRequests.Notifications/Tasks/SupportRequestsNotificationScheduledTask.cs]: This property is completely unused and caused a warning

File: tests/StatusAggregator.Tests/Update/AggregationEntityUpdaterTests.cs [hunk 3]
```diff
@@ -26,7 +26,7 @@ public class TheEventUpdateAsyncMethod
         {
         }
 
-        public class TheUpdateAsyncMethod<TChildEntity, TAggregationEntity>
+        public abstract class TheUpdateAsyncMethod<TChildEntity, TAggregationEntity>
```
Comment (loic-sharma) [inline tests/StatusAggregator.Tests/Update/AggregationEntityUpdaterTests.cs]: The visual studio test runner was trying to run `TheUpdateAsyncMethod<TChildEntity, TAggregationEntity>` when really it should've been running just `TheIncidentGroupUpdateAsyncMethod` and `TheEventUpdateAsyncMethod`. The CLI runner worked as expected, hence how this got checked in.

Comment (scottbommarito) [reply tests/StatusAggregator.Tests/Update/AggregationEntityUpdaterTests.cs]: The `abstract` shouldn't be necessary (I have this pattern in a couple places and never saw issues anywhere else) but I have no problem with putting it there.

Comment (loic-sharma) [reply tests/StatusAggregator.Tests/Update/AggregationEntityUpdaterTests.cs]: I tried removing the `abstract`, the tests appeared (and then failed) on VS's test explorer again. It seems that the `abstract` keyword is necessary to coax xunit to skip these tests.

**Pull request merged**